Prace_domowe/Praca_domowa4/Grupa1/Kacper_Grzymkowski/PD4.ipynb

```diff
@@ -0,0 +1,2439 @@
+{
```

Comment:
<p>Z uwagi na to, że nie użyto Pipeline nastąpił wyciek danych w czasie kroswalidacji. W każdej iteracji jest robiony podział na train i test czyli skalowanie danych mogło się odbyć z użyciem wartości, które trafiły do zbioru test. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>